ci/test_suites/authenticators_ldap/test

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# This is executed by the main "ci/test" script after cd-ing into "ci".
+# shellcheck disable=SC1091
+source "./shared.sh"
+
+_prepare_env_auth_ldap
+_run_cucumber_tests authenticators_ldap ldap-server
```

Comment:
I made this change but note: since we're not in a function we can't use "local".  Instead, this file is _executed_, which means that it runs in a subshell, which means it's variables are protected from leaking that way.